### ISSUE TITLE
Wrong provider in latest release

### ISSUE THREAD

**Issue by Bluedagger21** [dysath/seat-srp#48]
```
seat-web_1     | > Illuminate\Foundation\ComposerScripts::postAutoloadDump
seat-web_1     | > @php artisan package:discover --ansi
seat-web_1     |
seat-web_1     | In SrpServiceProvider.php line 8:
seat-web_1     |
seat-web_1     |   Class 'App\Providers\AbstractSeatPlugin' not found
seat-web_1     |
seat-web_1     |
seat-web_1     | Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

https://eve-seat.slack.com/archives/CAPC5V8QK/p1598864885029300

**Comment by Crypta-Eve** [comment]
Yeah, made a bad tag... My bad... Pushing fix asap

**Comment by Crypta-Eve** [comment]
Fixed in 4.0.2
